src/Tickets/Emails/Email/Completed_Order.php

```diff
@@ -46,23 +46,25 @@ class Completed_Order extends \TEC\Tickets\Emails\Email_Abstract {
 	public $template = 'admin-new-order';
 
 	/**
-	 * Email recipient.
+	 * Get email title.
 	 *
 	 * @since 5.5.10
```

Comment:
Should the `@since` be updated?

Reply:
This is git being weird, the `get_title` method was already there since 5.5.10

Reply:
The one introduced here is `get_to()` and it [has the TBD](https://github.com/the-events-calendar/event-tickets/pull/2596/files/aa13495876d18471122793083fa4c73645cb6287#diff-542651b7de9fb8fc1e79527bcbbf9828b7c3866e3b55c039a288a00687ab22fdR62)